.github/workflows/luacheck.yml

```diff
@@ -11,3 +11,11 @@ jobs:
         uses: actions/checkout@v3
       - name: Luacheck
         uses: lunarmodules/luacheck@v0
+
+      - name: Install EditorConfig Lint
+        uses: editorconfig-checker/action-editorconfig-checker@d4fca16fc71adef10fbe101903b654449fa9570c
```

Comment:
Why the commit hash? Don't they have a usable version tag? Preferably a GHA style major version tracking tag?

Reply:
They didn't seem to. `v1` was radically different to `main` which is what their documentation said to use.